packages/utilities/psammead-test-helpers/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-test-helpers",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
Shouldn't this be a patch release instead of a minor release?

Reply:
It's adding functionality, in the form of a new function, `testUtilityPackages`.

Reply:
👍